Friends at the end: Help Liam dig a grave for Chris, serves as a tutorial

I-am-Erk opened the pull request:
<!-- HOW TO USE: Under each "#### Heading" below, enter information relevant to your pull request.
You must have the below headings. Comments like this may be safely removed, if you want.

If you are opening this pull request from Github's web interface, you can use the 'preview' button to see what your pull request will look like to others.

Guidelines for pull requests:
-Keep your changes limited to one specific issue or change, plus the bare minimum related work to make that happen.
-A good rule of thumb is that most pull requests are less than 500 lines of changes.
-You can open extra pull requests to separate out portions of an intended change, ask if you're unsure. We're happy to work with you or advise the best way to get your PR merged.
-->

#### Summary
Content "Friends at the End: Dig a grave for your old friend Chris."

<!-- This section should consist of exactly one line, edit the one above.
1. Replace the word "Category" with one of these specific categories: Features, Content, Interface, Mods, Balance, Bugfixes, Performance, Infrastructure, Build, I18N.
2. Replace the text inside the quotes with a brief description of your changes.
Or if you don't want a changelog entry, replace the whole line with just the word "None" (with no quotes).
Some examples:
1. None
2. Features "In-game Armor sprite change"
3. Interface "Show crafting failure chances in the crafting interface"
For more on the meaning of each category, see:
https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/CHANGELOG_GUIDELINES.md
If merged, your summary will be added to the project changelog:
https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/changelog.txt -->

#### Purpose of change
Liam's scenario, Friends to the End, opens with him having de-zombified your mutual childhood friend Chris.  Liam has been waiting for closure on this for a few years.  My kid is now a huge Liam fan, and I gave them the choice for what content to add next, this is what they picked.

#### Describe the solution

This adds a series of missions which serve as a tutorial/intro to the game and also will have some nice charcater building for Liam, and possible the player. Also solves some minor bugs with Liam.

- [x] Quest 1: get a pair of shovels from the back room
- [x] Quest 2: dig a pit under the elm tree out back
- [ ] Quest 3: Drag Chris' body to the pit
- [ ] Quest 4: Construct an earthen grave for Chris
- [ ] Dialogue: Perform last rites for Chris by reminiscing about your past with Liam, and thereby getting to know your own character better

Infrastructure needed
- [x] Change Liam's backyard to have an elm tree to mark the spot where you should bury Chris
- [x] Add an 'earthen grave', a grave construction without a coffin or cairn, just a body in the ground and a marker
- [ ] Make Chris' body a unique item so that we can track it more easily for the quest

Adjacent fixes:
- Liam's car is now a unique vehicle named after my first car, the Rusty Duck.  It's still just a hatchback, but it never spawns with a security system (or a working heater). I may someday give it a unique skin.


#### Describe alternatives you've considered

I considered having this just be liam asking you to do the thing, and then an EOC allowing you to talk to Liam over the grave and perform last rites, but I think making it a quest is a good way to help Liam be a tutorial character. The quests should be quick enough that you can get it done pretty quickly.

#### Testing
Still in draft, working on it

#### Additional context
I considered adding some sequential time-based dialogue for Liam, or giving him more mission-specific stuff, but like I said, this is what my household Liam fankid requested, so here we are.

I-am-Erk committed:
start liam work

Give liam a unique car
Add first liam mission
Begin reworking mission dialogue
Get annoyed at how things have changed

I-am-Erk committed:
I'm digging a deep deep pit with my friends

PatrikLundell commented:
I was afraid that was the case. What about removing the panels over the wheels (damaged wheel houses, basically)?

I-am-Erk commented:
Honestly I'd be more inclined to add the code support to either pre-damage panels, or to mark a specific system as "never damage this one". Or, more within my wheel house, I might just make some custom art for the car that looks rusty.

I-am-Erk committed:
add earthen grave

I-am-Erk committed:
typo fix

I-am-Erk committed:
Update data/json/npcs/your_followers/liam_main_dialogue.json

I-am-Erk committed:
user just carrying a bunch of shovels around IG

I-am-Erk committed:
give liam shovel

Let's just track one. If you have two great, but if you only have one you just give it to Liam and have to go get the other. Then your backpack size doesn't come into play.

I-am-Erk committed:
Shovel check

I-am-Erk committed:
Merge branch 'master' into Friends-at-the-end

I-am-Erk committed:
Update cars.json

I-am-Erk committed:
Update liam_missions.json

I-am-Erk committed:
start adding mission params to mission 2

I-am-Erk committed:
Update liam_missions.json

vestigial condition to ensure mission doesn't end itself, it should actually be ended by dialogue.
